app/views/status/index.html.erb

```diff
@@ -1,9 +1,32 @@
-<% content_for :title do %>Conjur CE Status<% end %>
+<% content_for :title do %>Conjur Status<% end %>
 
 <div class="row">
   <div class="col-md-8">
     <h1 class="page-title">Status</h1>
-    <p class="font-large">Your Conjur CE server is running!</p>
+    <p class="font-large">Your Conjur server is running!</p>
+    <h2>Security Check:</h2>
+    <p>
+      Does your browser show a green lock icon on the left side of the address bar?
+      <dl>
+        <dt>Green lock:</dt><dd>good, Conjur is secured and authenticated.</dd>
+        <dt>Yellow lock:</dt>
+        <dd>
+          ok, Conjur is secured but not authenticated. Send your
+          Conjur admin to the
+          <a href="https://conjur.org/tutorials/integrations/nginx.html">Conjur+TLS guide</a>
+          to learn how to get the green lock.
+        </dd>
+        <dt>Red broken lock or no lock:</dt>
+        <dd>
+          Conjur is insecure. Don't put any secrets
```

Comment:
What do you think about softening this a bit, maybe something like: `Conjur is running insecurely. This is fine for playing around with Conjur, but not for storing secrets you'd like to stay secret. Visit the ...`

or something similar.  As someone running this locally to see what it does, I don't feel like I need to setup Conjur securely.

Reply:
Updated